library-core/src/main/java/com/mikepenz/fastadapter/listeners/CustomEventHook.kt

```diff
@@ -21,12 +20,8 @@ abstract class CustomEventHook<Item : GenericItem> : EventHook<Item> {
      * @param viewHolder
      * @return
      */
-    fun getFastAdapter(viewHolder: RecyclerView.ViewHolder): FastAdapter<Item>? {
-        val tag = viewHolder.itemView.getTag(R.id.fastadapter_item_adapter)
-        return if (tag is FastAdapter<*>) {
-            tag as FastAdapter<Item>
-        } else null
-    }
+    fun getFastAdapter(viewHolder: RecyclerView.ViewHolder): FastAdapter<Item>? =
```

Comment:
With the changes, this is no longer used. If you want you can deprecate this and tell users to just use the helper from `FastAdapter` directly?